contrib/automation_tests/orbit_vulkan_layer.py

```diff
@@ -0,0 +1,27 @@
+"""
+Copyright (c) 2020 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+
+from absl import app
+
+from core.orbit_e2e import E2ETestSuite
+from test_cases.capture_window import MatchTracks, Capture
+from test_cases.connection_window import FilterAndSelectFirstProcess, ConnectToStadiaInstance
+
+"""
+Note that this test is based on Trata (rather then hello_ggp).
```

Comment:
Please add a brief description of the preconditions (game started with Vulkan layer deployed)

Reply:
Done.